client/components/LearnerViewDirective.js

```diff
@@ -71,6 +50,27 @@ tie.directive('learnerView', [function() {
               </div>
             </div>
           </div>
+          <div class="tie-question-ui">
+            <div class="tie-question-window">
+              <h3>Exercise {{exerciseNumber}}: {{title}}</h3>
```

Comment:
Can the internal properties of exercises be separated from the view presented to the user? Not all exercises will be "Exercise 1" (e.g., if they are stand-alone). Maintenance and use of an exerciseNumber for application logic seems fair, but requiring that property to be displayed to the user in a specific format appears to be a UI judgement call.